holoviews/tests/core/data/test_spatialpandas.py

```diff
@@ -301,3 +304,14 @@ def test_dict_dataset_add_dimension_values(self):
 
     def test_sort_by_value(self):
         raise SkipTest("Not supported")
+
+
+
+def test_regression_get_value_array():
```

Comment:
Should this test run if spatialpandas is not installed?

Reply:
I don't think it hurts, as the test contains no spatialpandas code.